app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/websocket/Message.kt

```diff
@@ -7,7 +7,8 @@ data class Message(
     @SerializedName("value") private val value: String? = null,
     @SerializedName("offerSDP") val sdpOffer: SdpData? = null,
     @SerializedName("answerSDP") val sdpAnswer: SdpData? = null,
-    @SerializedName("baby_name") val babyName: String? = null
+    @SerializedName("baby_name") val babyName: String? = null,
+    @SerializedName("iceCandidate") val iceCandidate: IceCandidateData? = null
 ) {
     fun action() =
         if (action != null && value != null)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/8ce941c6c7cc4d13bad0e73c61419fa5d1ca7b0f/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/websocket/Message.kt#L14"></span>Multi-line if statement was found that does not have braces. These should be added to improve readability.</td>
    </tr>
  </tbody>
</table>